src/transformers/models/data2vec/ema_module.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# TODO: Remove file
```

Comment:
This was copied from fairseq's codebase to help in the conversion of model weights and should be removed before merging this PR

Reply:
think we can delete it now

Reply:
I deleted all the fairseq files